src/layouts/footer/index.tsx

```diff
@@ -19,7 +19,9 @@ const Footer: React.FC = () => {
 				<div className={styles.footer__centerBlock}>
 					<ul className={styles.footer__text}>
 						<li className={styles.footer__list_disabled}>О нас</li>
-						<li className={styles.footer__list_disabled}>Условия доставки</li>
+						<Link to={URLS.DELIVERY_COND} className={styles.footer__link}>
```

Comment:
Использование <Link> компонента внутри <ul> - нестандартное использование, рекомендуется перенести <Link> за пределы списка. Например:

```suggestion
						<li className={styles.footer__list}>
            						<Link to={URLS.DELIVERY_COND} className={styles.footer__link}>
																		Условия доставки
												</Link>
						</li>;
```